external-crates/move/crates/move-compiler/src/editions/mod.rs

```diff
@@ -279,6 +281,7 @@ impl FeatureGate {
             FeatureGate::NoParensCast => "'as' without parentheses is",
             FeatureGate::TypeHoles => "'_' placeholders for type inference are",
             FeatureGate::Lambda => "lambda expressions are",
+            FeatureGate::InlineModule => "inline 'module' forms (ending with ';') are",
```

Comment:
Is inline the right name here? 
Not sure I have a better one, but maybe something like 
`"'module' label forms (ending with ';') are"`

Reply:
+1 to the naming. What if we just call them "module labels" or something similar?